chain_linux.go

```diff
@@ -0,0 +1,114 @@
+package netlink
+
+import (
+	"fmt"
+	"github.com/vishvananda/netlink/nl"
+	"golang.org/x/sys/unix"
+)
+
```

Comment:
would it make sense to add Unit test for this file ?

Reply:
tests added